templates/netdev.epp

```diff
@@ -31,9 +31,6 @@ FirewallMark=<%= $firewall_mark %>
 <% $peers.each |$peer| { -%>
 
 [WireGuardPeer]
-<% if $peer['description'] { -%>
-Description=<%= $peer['description'] %>
```

Comment:
Removing description for peers looks good to me, but it might actually be useful to keep including it as comments (so it can be seen when looking at the config file).